plugins/basic/python/tk_nuke_basic/plugin_bootstrap.py

```diff
@@ -156,14 +155,8 @@ def __get_engine_name():
     engine_name = os.environ.get("SHOTGUN_ENGINE")
     if not engine_name:
         # FIXME: We are assuming that the configuration inside the standalone plugin
-        # has an engine instance with these name. These values should probably come
-        # from the plugin's info.yml file.
-        try:
-            import hiero # noqa
-        except:
-            engine_name = "tk-nuke"
-        else:
-            engine_name = "tk-nukestudio"
+        # has an engine instance with this name.
```

Comment:
There is only one section in the config at the moment, so the engine to bootstrap is always tk-nuke.